src/java.desktop/share/classes/sun/swing/text/TextComponentPrintable.java

```diff
@@ -781,7 +781,7 @@ private void calculateRowsMetrics() {
                             && (y != previousY || height != previousHeight)) {
                         /*
                          * we do not store the same value as previous. in our
-                         * documents it is often for consequent positions to have
+                         * documents it is often for consecutive positions to have
                          * the same modelToView y and height.
```

Comment:
If we are going to fix that, we might as well fix the entire sentence.
In particular I strongly suspect that the author meant "common" not "often".
"often" would only be correct here if used in a phrase like "often the case", but "common" is simpler.
And the usage of "previous" isn't right.
 I'm not sure I get what the commenter means to say here, perhaps "previously" ?

And capitalise "We", and "In" since they begin sentences

    * We do not store the same value as previously.
    * In our documents it is common for consequent positions to have
    * the same modelToView y and height.

Reply:
> And capitalise "We", and "In" since they begin sentences

I agree, yet it's not always followed in the comments.

> If we are going to fix that, we might as well fix the entire sentence.

Thank you for your suggestion. The sentence didn't sound right to me. Yet I couldn't figure out the better way to phrase it even though I tried re-writing it.